runtime/dotnet/azurewebapp/Startup.cs

```diff
@@ -168,6 +169,9 @@ public void ConfigureServices(IServiceCollection services)
             var resourceExplorer = new ResourceExplorer().AddFolder(botDir);
             var rootDialog = GetRootDialog(botDir);
 
+            // register customized types, please note this is only needed for customized actions. 
+            resourceExplorer.RegisterType<MultiplyDialog>("MultiplyDialog");
```

Comment:
@luhan2017 so currently the default runtime that is ejected out is going to include this registration for all samples correct? Do you think if we could include this registration only if they eject out CustomActions sample?

Reply:
yes we should include it for everyone so they have a clear example of how it works.